modules/widgets/twitter-timeline.php

```diff
@@ -152,29 +149,13 @@ public function widget( $args, $instance ) {
 		 */
 		$dnt = apply_filters( 'jetpack_twitter_timeline_default_dnt', false );
 		if ( true === $dnt ) {
-			echo ' data-dnt="true"';
+			$data_attrs .= ' data-dnt="true"';
 		}
 
 		if ( ! empty( $instance['chrome'] ) && is_array( $instance['chrome'] ) ) {
-			echo ' data-chrome="' . esc_attr( join( ' ', $instance['chrome'] ) ) . '"';
+			$data_attrs .= ' data-chrome="' . esc_attr( join( ' ', $instance['chrome'] ) ) . '"';
 		}
 
-		$type      = ( isset( $instance['type'] ) ? $instance['type'] : '' );
-		$widget_id = ( isset( $instance['widget-id'] ) ? $instance['widget-id'] : '' );
-		switch ( $type ) {
-			case 'profile':
-				echo ' href="https://twitter.com/' . esc_attr( $widget_id ) . '"';
-				break;
-			case 'widget-id':
-			default:
-				echo ' data-widget-id="' . esc_attr( $widget_id ) . '"';
-				break;
-		}
-		echo ' href="https://twitter.com/' . esc_attr( $widget_id ) . '"';
```

Comment:
Much of this PR's diff is simply moving logic to different lines, and changing `echo` to `$output .=`.

For example, this snippet is simply moved lower in the file, with `echo` changed to `$output .=`